docs/src/storage.md

```diff
@@ -51,65 +51,65 @@ we recommend that you also consider this aspect when you choose your storage
 solution, especially if you manage very large databases.
 
 !!! Important
-    Please refer to the official Kubernetes documentation for a list of all
-    the supported [Container Storage Interface (CSI) drivers](https://kubernetes-csi.github.io/docs/drivers.html)
-    that provide snapshotting capabilities.
+    See the Kubernetes documentation for a list of all
+    the supported [container storage interface (CSI) drivers](https://kubernetes-csi.github.io/docs/drivers.html)
+    that provide snapshot capabilities.
 
 ## Benchmarking CloudNativePG
 
-We recommend that you benchmark CloudNativePG in a controlled Kubernetes
-environment, before deploying the database in production, by following
-the [guidelines in the "Benchmarking" section](benchmarking.md).
+Before deploying the database in production, we recommend that you benchmark 
+CloudNativePG in a controlled Kubernetes environment. Follow
+the guidelines in [Benchmarking](benchmarking.md).
 
-Briefly, our advice is to operate at two levels:
+Briefly, we recommend operating at two levels:
 
-- measuring the performance of the underlying storage using `fio`, with relevant
+- Measuring the performance of the underlying storage using fio, with relevant
```

Comment:
We do not use tagging on names of tools, but we do on names of commands. In this case, I think you mean the tool. The fio website doesn't use any special tagging. So If you want to change it to "the `fio` command, the backticks are okay. Otherwise, please do not include the backticks.